src/Utils/navigationIcons.js

```diff
@@ -0,0 +1,155 @@
+import {
+  Activity16,
+  AddAlt16,
+  Analytics16,
+  Api16,
+  Apps16,
+  ChartScatter16,
+  FlowData16,
+  Fork16,
+  Chat16,
+  Code16,
+  Document16,
+  Home16,
+  TrashCan16,
+  Download16,
+  Edit16,
+  Filter16,
+  Idea16,
+  Launch16,
+  Locked16,
+  Power16,
+  Restart16,
+  ReportData16,
+  Rocket16,
+  Search16,
+  Settings16,
+  SettingsAdjust16,
+  Task16,
+  Terminal20,
+  Upload16,
+} from "@carbon/icons-react";
+
+/**
+ * this contant file is used to map icon names to render the icon. Used by the navigation menu.
+ */
+export const navigationIcons = [
```

Comment:
I think this includes some icons from another part of the application? I did implement things slightly different in CICD if you want to look at that for a reference.